kernel/prepare/pkg.yaml

```diff
@@ -11,42 +11,21 @@ steps:
         sha512: "{{ .linux_sha512 }}"
     env:
       ARCH: {{ if eq .ARCH "aarch64"}}arm64{{ else if eq .ARCH "x86_64" }}x86_64{{ else }}unsupported{{ end }}
+  - network: default
     prepare:
       - |
         tar -xJf linux.tar.xz --strip-components=1
         rm linux.tar.xz
 
-        mkdir /bin
-        ln -sv /toolchain/bin/bash /bin/bash
-        ln -sv /toolchain/bin/bash /bin/sh
-        ln -sv /toolchain/lib /lib
-        mkdir -p /usr/bin \
-            && ln -sf /toolchain/bin/env /usr/bin/env \
-            && ln -sf /toolchain/bin/true /bin/true \
-            && ln -sf /toolchain/bin/false /bin/false \
-            && ln -sf /toolchain/bin/pwd /bin/pwd
-
-        # Ensure that `make menuconfig` works.
-        ln -s /toolchain/bin/awk /usr/bin/awk
-
-        mkdir -p /usr/lib/pkgconfig
-        ln -s /toolchain/include /usr/include
-
         make mrproper
       - |
-        cd /toolchain && git clone https://github.com/a13xp0p0v/kernel-hardening-checker.git
+        mkdir -p /toolchain/kernel-hardening-checker
+        cd /toolchain/kernel-hardening-checker
+        curl -L https://github.com/a13xp0p0v/kernel-hardening-checker/archive/refs/heads/master.tar.gz | tar xz --strip-components=1
```

Comment:
Downloading this without any tag/hash lock has an advantage of bringing new tests in as soon as possible, however drawbacks are that this is a script being executed in kernel build process right before the kernel build (so if it's compromised it poses a security risk) and older builds (and thus configs) might potentially regress as time passes, due to new checks not accepting old configs

Reply:
right, we probably need a way to separate out `test` so that it doesn't affect steps (I can look into that).

Reply:
I can do that. However what about building older kernel configs in time when there's a breaking change in KSPP checker?

Reply:
yeah, we can pin it as we discussed in Slack